goon/code/datums/browserOutput.dm

```diff
@@ -4,6 +4,7 @@ For the main html chat area
 
 //Precaching a bunch of shit
 GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of icons for the browser output
+GLOBAL_VAR_INIT(emojiJson, file2text("goon/browserassets/js/emojiList.json"))
```

Comment:
Use `SSstrings`

Reply:
How do you suggest SSstrings for a .json file? I just don't know.

Reply:
> Use `SSstrings`

How do you suggest SSstrings for a .json file? I just don't know.

Reply:
its not in the strings directory so they can't.